query_optimizer/resolver/Resolver.cpp

```diff
@@ -143,6 +145,45 @@ namespace E = ::quickstep::optimizer::expressions;
 namespace L = ::quickstep::optimizer::logical;
 namespace S = ::quickstep::serialization;
 
+namespace {
+
+static attribute_id GetAttributeIdFromName(
```

Comment:
We don't need `static` key word for methods in an anonymous namespace.

Ditto for below.